### ISSUE TITLE
Make Environment Variables more secure

### ISSUE THREAD

**Issue by Zerotask** [openode-io/openode-www#326]
At the moment, you're able to see the clear text values of each environment variables. This can be a risk, if someone gets access to the account. Those data are usually very sensitive and therefore it would be more secure to remove the option to see this data.

In GitHub, once you added a secret (env variable) you can't read it, you can just override (update) or remove it. This means, even if someone gets access to my GitHub account, he won't be able to get those (very) sensitive data.

## Currently
![image](https://user-images.githubusercontent.com/20150243/122644147-17931b80-d114-11eb-95fd-222a61bfd96c.png)

## Comparison to GitHub (secrets)
![image](https://user-images.githubusercontent.com/20150243/122644169-309bcc80-d114-11eb-98e7-912c5bd53055.png)

**Comment by martinlevesque** [comment]
Sounds like a good suggestion, thanks!

**Comment by Zerotask** [comment]
Thanks for your update. It's better now.

Just 2 little additions:
- don't show the length of the variable content (improves security a bit)
- buttons: I'd change `Save` to `Update` and to be consistent, add a word for the delete button

### Before
![image](https://user-images.githubusercontent.com/20150243/123525136-f3fd4180-d6ce-11eb-9692-deccc10c812d.png)


### After
![image](https://user-images.githubusercontent.com/20150243/123525164-13946a00-d6cf-11eb-8aab-6938cae8aa92.png)

**Comment by martinlevesque** [comment]
Thanks for the feedback

**Comment by Zerotask** [comment]
I noticed a little bug now:

I created a new env variable test with the content test. Then I clicked on the update button, without inserting a new value:

![image](https://user-images.githubusercontent.com/20150243/123628154-1304ec00-d813-11eb-9620-913a58db1f3f.png)

Therefore there should be a simple validation, which checks if the input length is greater than 0. Otherwise the update button can also be disabled.
